dispatcher/queue_test.go

```diff
@@ -0,0 +1,49 @@
+package dispatcher
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/newscred/webhook-broker/storage/data"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPriorityQueueBasic(t *testing.T) {
+	contentType := "application/json"
+	pq := NewJobPriorityQueue()
+	for _, p := range []int{2, 3, 1} {
```

Comment:
Could we use a larger array for the test here?

Reply:
Done in https://github.com/newscred/webhook-broker/pull/134/commits/fafe0624300dfca2b949071e0a9790b9b8d3ba4b